eruditus/eruditus.py

```diff
@@ -107,15 +107,22 @@ async def create_ctf(self, name: str, live: bool = True) -> Union[dict, None]:
             role: discord.PermissionOverwrite(read_messages=True, send_messages=False),
         }
 
+        bot_cmds_channel = await guild.create_text_channel(
+            name="🤖-bot-cmds",
+            category=category_channel,
+            overwrites={
+                guild.default_role: discord.PermissionOverwrite(read_messages=False),
+                role: discord.PermissionOverwrite(
+                    read_messages=True, send_messages=True
+                ),
```

Comment:
`send_messages` is already implicitly set to `True` (all guild members can technically write to that channel, but since they can't view it in the first place, the permission is useless).
```suggestion
                role: discord.PermissionOverwrite(read_messages=True),
```